core/src/xtdb/tx.clj

```diff
@@ -464,6 +464,19 @@
       :else
       true)))
 
+;; introduced as a speculative change for 1.22 to avoid issues on ingest see #1800
+(def tx-batch-preferred-doc-count
```

Comment:
good to make this (temporarily) configurable - but let's pop it in as `::sys/args` on the ingester (can pair up)